Compilation fails with MLNX_OFED 5.0

This is reported by @ajameson:
```
libspead2.a(common_ibv.o): In function spead2::ibv_mr_t::ibv_mr_t(spead2::ibv_pd_t const&, void*, unsigned long, int)': common_ibv.cpp:(.text+0x828): undefined reference to ibv_reg_mr'
```
The problem is that RDMA-Core now defines `ibv_reg_mr` as a [macro](https://github.com/linux-rdma/rdma-core/blob/7af01c79e00555207dee6132d72e7bfc1bb5485e/libibverbs/verbs.h#L2426), which prevents the loader from functioning correctly.

--- a/gen/gen_ibv_loader.py
+++ b/gen/gen_ibv_loader.py
@@ -132,6 +132,10 @@ HEADER = PREFIX + '''\
 #include <infiniband/verbs.h>
 #include <rdma/rdma_cma.h>
 
+{% for node in nodes -%}
+#undef {{ node.name }}
+{% endfor %}
+
 namespace spead2
 {
 

